src/applications/check-in/travel-claim/app-entry.jsx

```diff
@@ -0,0 +1,16 @@
+import 'platform/polyfills';
+import '../sass/check-in.scss';
+
+import startApp from 'platform/startup';
```

Comment:
we should see if using the new import methods work for this app.

Reply:
I think we would have to refactor our router a bit for this to work. I got stuck trying.

Reply:
Ah, fair enough!